### ISSUE TITLE
leftShift(Closure)method has been deprecated and is scheduled to be removed in Gradle 5.0

### ISSUE THREAD

**Issue by shenliuyang** [ewerk/gradle-plugins#65]
gradle version 3.2

**Comment by griffio** [comment]
Thanks for pointing out that recent deprecation.

Referenced by https://github.com/gradle/gradle/issues/715

Looks like the recommended replacement is to use the ```doFirst``` and ```doLast``` methods.

@shenliuyang You are welcome to post a pull request if you want to make the change yourself

**Comment by griffio** [comment]
With gradle 3.2.1 or higher
```./gradlew wrapper --gradle-version=3.2.1```  
```./gradlew --stacktrace```                                                                                                                                                                                                                         

The Task.leftShift(Closure) method has been deprecated and is scheduled to be removed in ```
```
Gradle 5.0. Please use Task.doLast(Action) instead.
        at org.gradle.api.internal.AbstractTask.leftShift(AbstractTask.java:505)
        at org.gradle.api.Task$leftShift.call(Unknown Source)
        at net.researchgate.release.ReleasePlugin.apply(ReleasePlugin.groovy:70)
```

**Comment by holgerstolzenberg** [comment]
Fixed with 5a5835c881b452462de27a5a2829c293c4b402b5. Not released yet.

**Comment by shenliuyang** [comment]
thanks
